cobbler.spec

```diff
@@ -458,5 +459,3 @@ fi
 %{_datadir}/cobbler/tests/*
 
 %changelog
-* Thu Dec 19 2019 Neal Gompa <ngompa13@gmail.com>
```

Comment:
I think we can't remove this due to Redhatish Distros. So I would love if we could let this stay.

Reply:
I would be fairly upset if my changelog entry was erased, too.

Reply:
@SchoolGuy If this changelog stub is needed for Redhat build systems..., I agree. This was just an attempt to clean things further up. Sigh, if you clean it up on the one side, things fail on the other and vice versa...

Reply:
@Conan-Kudo This is the only changelog entry?!?
Changelog seem to be managed via .changes file and git history.
You probably have your comment still showing up in every rpm in the changelog.
I agree if this is not possible, if things do not build on Redhat based build services..., but otherwise I would be somewhat disappointed, if you insist of having the last and only changelog in an unused, out-of-date spec tag..